tests/animal_module_tests/test_animal.py

```diff
@@ -2207,7 +2209,7 @@ def test_calc_rqmts() -> None:
         "NEl_requirement": 1,
         "MP_requirement": 1,
         "Ca_requirement": 1,
-        "P_requirement": 1,
+        "P_requirement": 1 * (1 + (AnimalBase.config["ration"]["phosphorus_requirement_buffer"] / 100)),
```

Comment:
```suggestion
        "P_requirement": 1.35,
```
It is beneficial to hardcode expected results as much as possible in tests, because it will minimize the chances that an error in the logic being tested is carried over to the logic testing it.